zcnbridge/bridge.go

```diff
@@ -275,13 +275,16 @@ func (b *BridgeClient) BurnWZCN(ctx context.Context, amountTokens uint64) (*type
 	}
 
 	// 1. Data Parameter (amount to burn)
-	clientID := DefaultClientIDEncoder(b.ClientID())
+	//clientID := DefaultClientIDEncoder(b.ClientID())
+	//clientID = []byte(hex.EncodeToString(clientID))
+	//fmt.Println(b.ClientID())
+	//return nil, nil
 
 	// 2. Data Parameter (signature)
 	amount := new(big.Int)
 	amount.SetInt64(int64(amountTokens))
 
-	bridgeInstance, transactOpts, err := b.prepareBridge(ctx, "burn", amount, clientID)
+	bridgeInstance, transactOpts, err := b.prepareBridge(ctx, "burn", amount, b.ClientID())
```

Comment:
Please use the same data type of ClientID for gas estimation and Brun. Seems the previous code was correct.